spec/requests/api/v1/users/index_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Api::V1::UsersController, "#index", type: :request do
+  describe "list all users" do
+    let!(:users_list) { FactoryBot.create_list(:user, 5) }
+
+    context "when not authenticated" do
+      before(:each) do
+        get "/api/v1/users"
```

Comment:
```suggestion
        get "/api/v1/users", as: :json
```

To get formatted response so that you don't need parse json manually

Reply:
Thanks for this suggestion :)

Reply:
@tachyons `as: :json` gives `response.body` as `string` that would anyway be required to be decoded/parsed..
What's your thought ?

Reply:
It will be `response.parsed_body`